addons/craft/craft.lua

```diff
@@ -779,6 +779,37 @@ local function handle_command(cmd, ...)
     end
 end
 
+-- This is here so if a player does a manual synth the result is not displayed twice, since results are only hidden on injected synthesis.
+windower.register_event('outgoing chunk', function(id,original,modified,injected,blocked)
+	if id == 0x096 and injected then
+		injected_synth = true
+	end
+end)
+
+windower.register_event('incoming chunk', function(id,original,modified,injected,blocked)
+	if id == 0x06F and injected_synth then
+		local p = packets.parse('incoming',original)
+		if p['Result'] == (0 or 2) then
+			local item = res.items[p['Item']].en
+			windower.add_to_chat(121, 'You synthesized: \30\02%s\30\01.':format(item))
+		injected_synth = false	
+		end
+		if p['Result'] == (1 or 5) then
+			windower.add_to_chat(121,'Your synthesis has failed and your crystal is lost.')
+			if p['Lost Item 1'] ~= 0 then windower.add_to_chat(121, 'You lost: \30\02%s\30\01.':format(res.items[p['Lost Item 1']].en)) end 
```

Comment:
Use `.english` instead of `.en` and, if possible, use the `:color` function.

Reply:
fixed english. Tried to use :color, but couldn't find the right color(mentioned in discord)